client/rest-client/src/main/java/org/infinispan/client/rest/configuration/RestClientConfigurationBuilder.java

```diff
@@ -215,18 +216,20 @@ public RestClientConfiguration build(boolean validate) {
    }
 
    @Override
-   public RestClientConfigurationBuilder read(RestClientConfiguration template) {
+   public RestClientConfigurationBuilder read(RestClientConfiguration template, Combine combine) {
       this.connectionTimeout = template.connectionTimeout();
-      this.servers.clear();
+      if (combine.children() == Combine.Children.OVERRIDE) {
+         this.servers.clear();
+         this.headers.clear();
+      }
```

Comment:
Differently from other cases where we apply the Children override strategy we don't check here if the `template.attributes().isTouched()`. Is that any reason for that?

Reply:
Because the template does not use attributes.

Reply:
Ok makes sense thank you